src/app/dynamic-forms/fields/child-form-table/child-form-table.component.html

```diff
@@ -49,7 +49,13 @@
       </tbody>
     </table>
   </div>
-  <p *ngIf="!childForm.childFormRowValues?.length">
-    There is no data in this table. You can use the add button above to get started.
+  <p *ngIf="childForm.childFormLinkId == null || childForm.childFormParentId == null ;else noRecord"> 
+     The columns "Child Form Link Field" or both the “Child Form Parent ID” and the "Child Form Link ID" must be populated in your form fields table. 
   </p>
```

Comment:
The quotes around "Child Form Parent ID" are different, this usually happens when copy/pasting.  We should maybe change the quotes to be the same as others.

Reply:
Hi @cysonya , I have resolved above comment.